_posts/2024-03-11-herzlich-willkommen-bei-holiday-check.md

```diff
@@ -0,0 +1,126 @@
+    <figcaption>Image showing Presidio</figcaption>
+</figure>
+
+
+**Step 3:** Once the transcription process is completed and passed through the privacy scans, a summary of the call is generated by sending the script to a local inference server for "mistral-7b-instruct-v0.2.Q8_0.gguf". The inference server is built on top of [llama.cpp](https://github.com/ggerganov/llama.cpp).
+During this step, we also translate the German transcripts into English transcripts - this makes them accessible to even more wider audiences, further scaling-up product and customer service.
+
+
+<blockquote>
+
+SUMMARY
+
+The conversation between the HolidayCheck Agent and the customer revolved around the Holiday Check Premium Club membership and its associated benefits, particularly the use of Gutschein (vouchers) for travel and car rental bookings. 
+Following are the points discussed were as follows:
+
+1. The customer was on HolidayCheck's booking form and had come across a reference to a club membership and related vouchers.
+2. Sabine Müller, the agent, confirmed that they have the Holiday Check Premium Club, where members can secure numerous advantages for their travel arrangements, including two vouchers: one worth €50 when booking a trip valued at €490 or more, and a €100 voucher when booking one priced at €990 or above. They also receive car rental discounts and premium price reductions.
+3. The customer understood that these discounts would effectively make their travel or car rental costs lower after they have taken the trip and presented their membership or vouchers.
+4. To avail of the voucher benefits, customers first book a trip or rental normally and then submit their membership details or vouchers. After returning from their trip, they will receive the corresponding voucher amount back as a refund.
+5. The discounts apply to both travel and car rentals.
+6. The size of the voucher depends on the total cost of the rental; there are four tiers based on €100, €300, €500, and €1000 Euro rental costs.
+7. The customer can learn more about these benefits by visiting the HolidayCheck website or subscribing to their newsletter.
+8. The membership lasts for one year but can be cancelled before renewal with a simple email notification to HolidayCheck.
+9. Customers pay for the membership online and receive an invoice, which they can settle through bank transfer.
+10. They may join at any time during the year, and the benefits apply from that point onwards, even if they book their first trip before joining.
+11. The agent confirmed that customers do not need to provide bank information for membership sign-up or voucher redemption. Instead, they can pay online and receive an invoice for settlement.
+12. Customers can join the Premium Club at any time, and the benefits are valid indefinitely from the date of joining, with no specific time restrictions.
+Even long talks can be understood in just a little over a minute. That's pretty fast!*
```

Comment:
I would also add a section about evaluating the outcome and how we keep track that the summary/translation is good enough

Reply:
Addes as "step 4"